axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,212 @@ def strategy(self, opponent: Player) -> Action:
+            Set FD = 0                                                                                                                                                                                                                                                  
+            Set the variable 'NO':                                                                                                                                                                                                                                      
+            NO = ((NO * NK) + 3 - (3 * J) + (2 * JA) - (JA * J) / (NK + 1)                                                                                                                                                                                              
+            Where JA and J are the last moves of the strategy and the opponent (C=0, D=1)                                                                                                                                                                       
+            Increase the value of NK by 1                                                                                                                                                                                                                               
+            Cooperate                                                                                                                                                                                                                                                   
+                                                                                                                                                                                                                                                                        
+    Tranquilizer came in 27th place in Axelrod's second torunament. 
+   
+
+    Names:
+
+    - Tranquilizer: [Axelrod1980]_
+    """
+
+    name = 'Tranquilizer'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': {"game"},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+
+    # Initialised atributes
+    def __init__(self):
+        super().__init__()
+        self.num_turns_after_good_defection = 0
+        self.consecutive_defections = 0
+        self.one_turn_after_good_defection_ratio= 5
+        self.two_turns_after_good_defection_ratio= 0
+        self.one_turn_after_good_defection_ratio_count = 1
+        self.two_turns_after_good_defection_ratio_count = 1
+        self.score = None
+        self.probability = 1.1
```

Comment:
Why this initial value of `probability`? Looking at the source code I don't see where this comes from.

Reply:
Initially, the value of `probability` was just `float`, however, this caused the code to throw up some errors due bug within the `update_state`. These bugs have now been rectified and the value of `probability` has now been reverted back to `float`.